ee/ucp/kubernetes/storage/use-csi.md

```diff
@@ -47,11 +46,10 @@ managing the cluster-wide logic and `DaemonSets` for managing node-specific logi
 
 You can deploy multiple CSI drivers for different storage backends in the same cluster.
 
-**Note**: 
-
-- To avoid credential leak to user processes, Kubernetes recommends running CSI Controllers on master nodes and the CSI node plugin on worker nodes. 
-- UCP allows running privileged pods. This is needed to run CSI drivers.
-- The Docker daemon on the hosts must be configured with Shared Mount propagation for CSI to allow the sharing of volumes mounted by one container into other containers in the same pod or even to other pods on the same node. By default, Docker daemon in UCP enables "Bidirectional Mount Propagation".
+> Note
+> - To avoid credential leak to user processes, Kubernetes recommends running CSI Controllers on master nodes and the CSI node plugin on worker nodes. 
+> - UCP allows running privileged pods. This is needed to run CSI drivers.
+> - The Docker daemon on the hosts must be configured with Shared Mount propagation for CSI to allow the sharing of volumes mounted by one container into other containers in the same pod or even to other pods on the same node. By default, Docker daemon in UCP enables "Bidirectional Mount Propagation".
```

Comment:
Suggest we break down this sentence. How about: 
`The Docker daemon on the hosts must be configured with Shared Mount propagation for CSI. This is to allow the sharing of volumes mounted by one container into other containers in the same pod or even to other pods on the same node.`